specification.markdown

```diff
@@ -14,24 +14,6 @@ Just like XML, the specific "tags" (IDs in EBML parlance) used in an
 EBML implementation are arbitrary. However, the semantic of EBML
 outlines general data types and ID's.
 
-The known basic types are:
```

Comment:
Note that moved the location of the section on data types to later in the specification.